jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/ui/viewmodel/HomePageViewModel.kt

```diff
@@ -42,6 +42,8 @@ class HomePageViewModel : ViewModel() {
     val translationModels = observableListOf<TranslationCardModel>()
     val resumeBookProperty = SimpleObjectProperty<Workbook>()
 
+    private val noResumableProject = -1
```

Comment:
This should be a const at the top of the file, and all caps with underscores NO_RESUMABLE_PROJECT